webapp/package.json

```diff
@@ -19,7 +19,8 @@
     "clean": "npm run clean --workspaces --if-present"
   },
   "dependencies": {
-    "react-intl": "6.5.5"
+    "react-intl": "6.5.5",
+    "react-zoom-pan-pinch": "3.4.3"
```

Comment:
Would you mind moving this to `webapp/channels/package.json` instead of here? We try to keep dependencies that are only for the web app there while this contains shared ones

Reply:
done